ledger/ledger-on-sql/src/app/scala/com/daml/ledger/on/sql/SqlLedgerFactory.scala

```diff
@@ -77,11 +75,6 @@ object SqlLedgerFactory extends LedgerFactory[ReadWriteService, ExtraConfig] {
         engine = engine,
         jdbcUrl = jdbcUrl,
         resetOnStartup = false,
-        logEntryIdAllocator = RandomLogEntryIdAllocator,
-        stateValueCache = caching.WeightedCache.from(
-          configuration = config.stateValueCache,
-          metrics = metrics.daml.kvutils.submission.validator.stateValueCache,
-        ),
```

Comment:
I think we need to put these lines back in.

Reply:
Ah yes, forgot to readd the built cache. Good catch